sdk/eventgrid/azure-eventgrid/azure/eventgrid/_version.py

```diff
@@ -9,4 +9,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "4.11.1"
+VERSION = "4.13.0"
```

Comment:
its jumps to 4.13 because 4.12 is taken by the beta ?

Reply:
yup!